src/test/kotlin/racingcar/RaceTest.kt

```diff
@@ -0,0 +1,25 @@
+package racingcar
+
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.Test
+
+class RaceTest {
+    @Test
+    fun `자동차 생성 테스트`() {
+        val race = Race(listOf("car1", "car2", "car3"))
+        assertThat(race.cars.size).isEqualTo(3)
+        race.cars.forEach { assertThat(it.status()).isEqualTo(0) }
+        race.cars.forEachIndexed { index, car -> assertThat(car.name).isEqualTo("car${index + 1}") }
+    }
+
+    @Test
+    fun `우승자 테스트`() {
+        val race = Race(listOf("car1", "car2", "car3"))
+        race.cars[0].move(4)
+        race.cars[1].move(4)
+        val winners = race.getWinner()
+        assertThat(winners.size).isEqualTo(2)
+        assertThat(winners.any { it.name == "car1" }).isTrue()
+        assertThat(winners.any { it.name == "car2" }).isTrue()
```

Comment:
```suggestion
         assertThat(winners).containsExactly("cars", "cars2")
```

containsExactly를 사용해 볼까요? 그리고 containsExactly는 장단점이 무엇이 있을까요? 그리고 단점이 있을 경우 대안이 되는 메서드는 또 무엇이 있을까요?